lib/main_common.pm

```diff
@@ -19,6 +19,7 @@ use testapi qw(check_var get_var get_required_var set_var check_var_array diag);
 use autotest;
 use utils;
 use version_utils qw(:VERSION :BACKEND :SCENARIO);
+use Utils::Backends 'is_remote_backend';
```

Comment:
you add the include but you don't seem to use the method in this file?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/fc52c6b3fecdbd7648b951ba3fa21ddfbe6903f8/lib/main_common.pm#L385